package.json

```diff
@@ -51,9 +52,17 @@
     "chai": "^4.1.1",
     "chai-as-promised": "^7.1.1",
     "chai-datetime": "^1.4.1",
+    "chalk": "^2.2.0",
```

Comment:
This version is ahead of the one [we've included in Desktop](https://github.com/desktop/desktop/blob/master/app/package.json#L22), and is likely the cause of these build errors.

Reply:
Can I just ⬆️ the one in `app/package.json`? It’ll let us 🔥 `@types/chalk`.

Reply:
Uh, sure